paper/paper.md

```diff
@@ -0,0 +1,337 @@
+Next, learners get a chance to put everything they have learned into
+practice by forming small groups and working on practice problems that cover the entire course
+content.
+The workshop completes with a short module recapping everything that the curriculum covered as well as offering suggestions on how
+learners can continue to get help and keep learning once the workshop ends.
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+Our modules and instructor suggestions are developed in the style of [Software Carpentry](https://software-carpentry.org/):
+
+1. Each module contains learning objectives at the beginning of each lesson and
+a summary of key points.
+1. The five core modules (2 to 6) are designed to be taught via live coding of
+the content to learners.
+This is a central feature of Carpentries lessons, and we believe it is a great
+way to learn how to program. It requires learners to follow along and
+encounter errors that they must debug along the way, fostering additional
+questions about the course content. It also leads to instructors making mistakes
+and then demonstrating how to deal with them in an ad hoc and iterative manner.
+1. We incorporate formative assessments in the form of short practice exercises
+throughout each lesson such that learners can practice what they have learned, while
+instructors can gauge learner understanding of the material.
+
+We also incorporate a few additional key components into the curriculum:
+
+1. Each lesson builds off of others, with the goal of creating a final report
+that can be shared with others.
+1. We structured the curriculum such that it could be taught through an
+in-person or virtual workshop. Virtual workshops are sometimes necessary, as during the COVID-19 pandemic, but are also useful to allow people from a variety of geographic locations to instruct and attend.
+1. We require learners to install all software before the workshop (as The
+Carpentries also requires), and ask them to run an example script that tests
+whether everything is installed correctly.
+Learners must send screenshots of the script output to the workshop lead in
+advance in order to attend. This ensures that any installation issues can be
+addressed before the workshop begins.
+1. An extensive small group practice module towards the end of the workshop
+allows learners to more independently practice the skills they have learned.
+1. The workshop concludes with a recap of what was covered and resources
+available for learners to continue learning and getting help as their skills
+develop.
+
+## Pilot Workshop
+<!-- experience of use -->
+
+We piloted our curriculum during a virtual two-day Software Carpentry workshop.
+In line with The Carpentries recommendations
+[@the_carpentries_carpentries_2018], we had four instructors and six helpers at
+the workshop to assist with learner questions and technical issues.
+We also used the "sticky note" system for formative assessment, where learners
```

Comment:
Maybe we should move this up to the section where we list what we use from The Carpentries?